src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -239,33 +239,46 @@ public PatternMatches GetMatches(string candidate, string dottedContainer)
         /// so, unless you need to know the full set of matches, use this version.
         /// </remarks>
         /// <param name="candidate">The word being tested.</param>
-        /// <param name="inludeMatchSpans">Whether or not the matched spans should be included with results</param>
+        /// <param name="includeMatchSpans">Whether or not the matched spans should be included with results</param>
         /// <returns>If this was a match, the first element of the set of match types that occurred while matching the
         /// patterns. If it was not a match, it returns null.</returns>
-        public PatternMatch? GetFirstMatch(string candidate, bool inludeMatchSpans = false)
+        public PatternMatch? GetFirstMatch(
+            string candidate, bool includeMatchSpans)
         {
             if (SkipMatch(candidate))
             {
                 return null;
             }
 
-            return MatchPatternSegment(candidate, inludeMatchSpans, _fullPatternSegment, wantAllMatches: false, allMatches: out _, fuzzyMatch: false) ??
-                   MatchPatternSegment(candidate, inludeMatchSpans, _fullPatternSegment, wantAllMatches: false, allMatches: out _, fuzzyMatch: true);
+            return GetFirstMatchWorker(candidate, includeMatchSpans, fuzzyMatch: false) ??
+                   GetFirstMatchWorker(candidate, includeMatchSpans, fuzzyMatch: true);
         }
 
-        private StringBreaks GetWordSpans(string word)
+        public PatternMatch? GetFirstMatch(
```

Comment:
:bulb: It seems like this method can be removed if we just update `TestGetFirstMatch` to coerce a fuzzy result to null. The current pair of overloads is difficult to understand because instinct tells a developer the one normally used is equivalent to the second except that it provides a default value for `fuzzyMatch`, yet this is not the case in practice.